extensions/stats/plugin.cc

```diff
@@ -179,6 +181,17 @@ void map(IstioDimensions& instance, bool outbound,
   }
 }
 
+// Conversion of a baggage string to workload labels string
+std::string baggageToLabels(std::string baggage) {
```

Comment:
Can we use baggage encoding in labels? Any parsing code is potentially dangerous, and it's easier to harden just one encoding.

Reply:
I agree but changing the host metadata labels to baggage encoding will make this PR scope broader touching with refactoring that will affect current Istio and Ambient.
I think that I can instead have a function in the using `WorkloadMetadataObject` that serializes it as labels encoding. In future PR we can make all baggage encoding.
Wdyt?

Reply:
SG.